src/main/resources/schema/provider.definition.schema.v1.json

```
@@ -85,6 +85,26 @@
                             "type": "boolean",
                             "default": true
                         },
+                        "nullEqualsEmpty": {
+                            "description": "When set to false, this flag indicates that null and empty list are not equal, and changing the values from null to empty list would indicate diff",
+                            "type": "boolean",
+                            "default": false
+                        },
+                        "caseInsensitive": {
```

thoughts on reversing this to `caseSensitive` instead? Might be more straightforward than specifying something as not `caseInsensitive`
```suggestion
                        "caseSensitive": {
```

---

valid point